tests/test_confidence.py

```diff
@@ -54,15 +54,49 @@ def test_confidence_leastsq(data, pars, verbose, capsys):
         assert 'Calculating CI for' in captured.out
 
 
+def test_confidence_pnames(data, pars):
+    """Test if pnames works as expected."""
+    minimizer = lmfit.Minimizer(residual, pars, fcn_args=(data))
+    out = minimizer.leastsq()
+
+    assert_paramval(out.params['a'], 0.1, tol=0.1)
+    assert_paramval(out.params['b'], -2.0, tol=0.1)
+
+    ci = lmfit.conf_interval(minimizer, out, p_names=['a'])
+
+
+def test_confidence_bounds_reached(data, pars):
+    """Check if conf_interval handles bounds correctly"""
+
+    # Should work
+    pars['a'].max = 0.2
+    minimizer = lmfit.Minimizer(residual, pars, fcn_args=(data))
+    out = minimizer.leastsq()
+    out.params['a'].stderr = 1
+    ci = lmfit.conf_interval(minimizer, out, verbose=True)
```

Comment:
did ```flake8``` not warn here about an assigned, but unused variable? Anyway, if you're not going to ```assert``` anything in the output, but just want to see if the function call works please just do ```lmfit.conf_interval(minimizer, out, verbose=True)```